clusterloader2/testing/l4lb/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{.Name}}
+  labels:
+    test: l4lb-load
+  annotations:
+    cloud.google.com/load-balancer-type: {{.LoadBalancerType}}
+spec:
+  externalTrafficPolicy: {{.ExternalTrafficPolicy}}
+  type: LoadBalancer
+  selector:
+    name: {{.DeploymentBaseName}}-{{.Index}}
+  ports:
+  - port: 8080
```

Comment:
I think that 8080 port on a node will conflict with NodeLocal DNSCache

Reply:
Good point, though it is only a problem if the pod is in "hostNetwork mode". This one is not.

Reply:
Isn't port and targetPort swapped then? service should be on 80 (port), pod exposes 8080 (targetPort)

Reply:
yes. Corrected the yaml. 

Now service and container port are both 80